### ISSUE TITLE
glossary: tooltips break details sections

### ISSUE THREAD

**Issue by jorgeorpinel** [iterative/dvc.org#510]
As a result of implementing #424, `<abbr>` tags now create glossary tooltips around certain terms in the docs.

However if one of these terms is placed inside a `<details>` section, the docs engine fails to create the expandable boxes (a pre-existing feature).

---
UPDATE: The example below is outdated but you get the idea hopefully.

For example open `static/docs/user-guide/external-dependencies.md` and change

```md
### Expand to learn more about DVC internals

The resulting "import stage" DVC-file will contain something like this:
```

which looks

![image](https://user-images.githubusercontent.com/1477535/62177182-3f4f1f80-b301-11e9-9fb3-5da46710726a.png)

for

```md
### Expand to learn more about DVC internals

The resulting <abbr>import stage</abbr> DVC-file will contain something like
this:
```

resulting in:

![image](https://user-images.githubusercontent.com/1477535/62177236-6f96be00-b301-11e9-94fb-74eba6e3927c.png)

**Comment by jorgeorpinel** [comment]
@algomaster99 do you want to take a look at this? 🙂

**Comment by algomaster99** [comment]
@jorgeorpinel This is happening since it only detects the `abbr` tags and just ignores the `details` tag. Seems weird to me. I'll have to check the inner functioning of HtmlRenderer.

**Comment by jorgeorpinel** [comment]
Any luck  with this first glossary-related issue, @algomaster99 ? Thanks 🙂

**Comment by jorgeorpinel** [comment]
- [ ] When fixed, we should add `<abbr>` to term "project" in the first `<details>` section of static/docs/get-started/configure.md, for now removed in ce3d940

**Comment by jorgeorpinel** [comment]
"Remember me @algomaster99" -This issue 🙂

**Comment by algomaster99** [comment]
@jorgeorpinel I think this would require some changes in `react-markdown` itself. I found one issue which is relevant - https://github.com/rexxars/react-markdown/issues/302

**Comment by fabiosantoscode** [comment]
I've opened this PR there which fixes this problem: https://github.com/rexxars/react-markdown/pull/386

**Comment by shcheklein** [comment]
@fabiosantoscode awesome, thanks.

**Comment by rogermparent** [comment]
With #1334, this seems to be fixed for me:
![dvc-abbr-desktop](https://user-images.githubusercontent.com/9111807/82840987-89158180-9ea2-11ea-8820-37e1fddc2bb8.png)
![dvc-abbr-mobile](https://user-images.githubusercontent.com/9111807/82840992-8a46ae80-9ea2-11ea-83bd-c53f6ccded0c.png)

This "When fixed" list may also need to update to our current docs content.

> * [ ]  We should add `<abbr>` to term "project" in the first `<details>` section of static/docs/get-started/configure.md, for now removed in [ce3d940](https://github.com/iterative/dvc.org/commit/ce3d940031e10a366298a24870557a5194af3d98).

Like this one, for example: we don't seem to have an glossary entry for "project"- did we change this term in the past?

On a related note, I made #1357 because the test page threw a null error when I clicked a `details` on mobile without the change.

Anyway, once others confirm this is really fixed I think we'll want to draft up a new list of `abbr`s to add inside `details` and either add that to this issue or close this and make a new content-only one. Up to everyone's preference, really.
